### PR TITLE
Style guide applied to Pages

### PR DESCRIPTION
**Pull request by cvereau**
Change Raking.js file to be an arrow function instead of a class since it's not managing the state. Dashboard it's not managing the state either but is using a component native function to do an initial load and fetch the data from the API, so it stays as component. 

Aditional refactor : Component instead of React.Component

### CODE REVIEW

**Comment by cvereau** [comment]
[1.1 Applying styles guides to Pages](https://trello.com/c/OEGTS1m8/24-1-1-applying-styles-guides-to-pages)

File: src/js/pages/Login.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Component } from 'react'
```
Comment (JohnProg) [inline src/js/pages/Login.js]: Add semicolon after each statement.

File: src/js/pages/Notification.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default class Notification extends React.Component {
   render () {
 
     const { activeItem } = this.state
-    const { list } = this.props.employee 
+    const { list } = this.props.employee
```
Comment (JohnProg) [inline src/js/pages/Notification.js]: Add semicolon.

**Review by orlando1409 — APPROVED** [review]
the code is aligned to code standards.

**Pull request merged**